### ISSUE TITLE
[Bug] [Beta] [Regression] dynamically appearing and disappearing tabs tray 3-dot menu

### ISSUE THREAD

**Issue by Neutrino9** [mozilla-mobile/fenix#13964]
Found while proposing UX changes in https://github.com/mozilla-mobile/fenix/issues/13963

## Steps to reproduce

* Open a website in Private browsing mode as well as Normal each
* change modes continuously
* Click on the 3-dot menu? (Which is not there, obviously)

### Expected behavior

Visible 3-dot menu button

### Actual behavior

Invisible (?) 3-dot menu button.

### Device information

* Android device: Motorola One Power.
*Android version: 10
* Fenix version: `80.0.1-beta.2`

![20200820_125522](https://user-images.githubusercontent.com/69946395/90729976-e0b36580-e2b6-11ea-9da2-38760cefc0eb.gif)

**Comment by astarq** [comment]
The menu is only shown when at least one tab is open, as all the options are related to tab actions. Did you try opening 1+ tabs and see if it appears?

**Comment by Neutrino9** [comment]
> The menu is only shown when at least one tab is open, as all the options are related to tab actions. Did you try opening 1+ tabs and see if it appears?

Yes, I definitely did. The 3-dot menu was disappearing even when the tabs were open in both PBM and normal browsing mode. That's why I gave the STR and said you need to open at least one tab in each mode. Seems like I recorded the wrong GIF, let me remove it as it'd cause confusion. Thanks for pointing that out.

**Comment by astarq** [comment]
Ooh yeah, I speed read it. Sorry.
